bookwyrm/static/js/bookwyrm.js

```diff
@@ -113,9 +113,44 @@ let BookWyrm = new class {
      * @return {undefined}
      */
     updateCountElement(counter, data) {
+        let count = data.count;
+        const count_by_type = data.count_by_type;
         const currentCount = counter.innerText;
-        const count = data.count;
         const hasMentions = data.has_mentions;
+        const allowedStatusTypesEl = document.getElementById('unread-notifications-wrapper');
+
+        // If we're on the right counter element
+        if (counter.closest('[data-poll-wrapper]').contains(allowedStatusTypesEl)) {
```

Comment:
I'm not completely happy with this way of discriminating between poll sites, but this is dependent on Django's ways of inserting JSON data into the code.